test/playground/routes.js

```diff
@@ -93,13 +105,17 @@ exports.open = function (req, res) {
         clientAreaSize: null
     };
 
+    // NOTE: We must store the 'browser' structure before the call to 'open' function, because sometimes the browser sends
+    // client size information before 'open' function resolves.
```

Comment:
@AndreyBelym just for beauty use the following formatting:
```js
// NOTE: We must save the 'browser' structure before we call the 'open' function, because 
// sometimes the browser sends client size information before the 'open' function resolves.
```